### PR TITLE
Identity v3 Projects Create

### PR DESCRIPTION
**Pull request by jtopjian**
For #157 
Depends on #164

### CODE REVIEW

**Comment by jtopjian** [comment]
@jrperritt This is ready for review.

Code has been rebased, acceptance and unit tests are passing. Coveralls is just complaining.

File: openstack/identity/v3/projects/results.go [hunk 1]
```diff
@@ -14,6 +14,16 @@ type GetResult struct {
 	projectResult
 }
 
+// CreateResult temporarily contains the reponse from the Create call.
+type CreateResult struct {
+	projectResult
+}
+
+// createErr quickly wraps an error in a CreateResult.
+func createErr(err error) CreateResult {
```
Comment (jrperritt) [inline openstack/identity/v3/projects/results.go]: Should this be removed? If not, what is its purpose?

Comment (jtopjian) [inline openstack/identity/v3/projects/results.go]: I assumed it was part of how errors are formed somehow, but that was just an assumption. It exists in the other identity packages, so I copied it here.

Comment (jrperritt) [inline openstack/identity/v3/projects/results.go]: I don't think they're used. Let's remove it.

Comment (jtopjian) [inline openstack/identity/v3/projects/results.go]: :+1: 

I'll remove this one now. I can open a separate PR for the others.

**Comment by jtopjian** [comment]
@jrperritt ping

Coveralls is now happy, too :)

**Comment by jrperritt** [comment]
aces

**Pull request merged**